packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -87,46 +101,123 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-.applications-list li {
-  margin: var(--oc-space-xsmall) 0;
+#applications-menu {
+  z-index: 3;
 
-  a,
-  button {
-    gap: var(--oc-space-medium);
-    justify-content: flex-start;
-    width: 100%;
+  .dropdown-title-wrapper {
+    position: relative;
+    height: 50px;
 
-    &.oc-button-primary-raw-inverse {
-      &:focus,
-      &:hover {
-        color: var(--oc-color-swatch-primary-contrast) !important;
-      }
+    h3 {
+      margin-left: 10px;
+      font-weight: 300;
+      position: absolute;
+      overflow: hidden;
     }
-    &.oc-button-passive-raw {
+  }
+}
+.applications-list {
+  width: 320px;
+  padding: 10px;
+  display: flex;
+  gap: 10px;
+  li {
+    margin: var(--oc-space-xsmall) 0;
+    border-radius: 13px;
+    display: inline-flex;
+    overflow: hidden;
+    gap: 10px;
+    a,
+    button {
+      display: inline-block;
+      gap: var(--oc-space-medium);
+      justify-content: flex-start;
+      width: 50px;
+      overflow: hidden;
+      text-align: center;
+      font-size: 12px;
+      padding: 0;
+      color: var(--oc-color-swatch-passive-default);
+
       &:focus,
       &:hover {
-        color: var(--oc-color-swatch-passive-default) !important;
+        color: var(--oc-color-swatch-passive-default);
+        text-decoration: none;
+        .icon-box {
+          box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+        }
       }
-    }
 
-    &:focus,
-    &:hover {
-      background-color: var(--oc-color-background-hover);
-      text-decoration: none;
-    }
+      .icon-box {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 50px;
+        height: 50px;
+        border-radius: 12px;
+        color: var(--oc-color-swatch-inverse-default);
+        margin-bottom: 2px;
+        transition: all 0.2s ease-in-out;
+      }
+      &:hover > .icon-box {
+        transform: scale(0.9);
+      }
 
-    .icon-box {
-      display: inline-flex;
-      justify-content: center;
-      align-items: center;
-      width: 40px;
-      height: 40px;
-    }
+      &.router-link-active {
+        color: var(--oc-color-text-default);
+        font-weight: 500 !important;
+        .icon-box {
+          box-shadow: inset 0 0 0px 2px var(--oc-color-text-default);
+        }
+      }
 
-    .active-check {
-      position: absolute;
-      right: 1rem;
+      .active-check {
+        position: absolute;
+        right: calc(1rem + 5px);
+      }
     }
   }
 }
+
+/* Application-Gradients */
+// TODO: Add default gradient / or add prop to choose gradient in the future
+.gradient-files {
```

Comment:
Having dedicated classes for the different apps is not an option. Doesn't scale. Also pretty difficult in theming.